stories/forms/inputs/textarea-story.js

```diff
@@ -36,3 +38,10 @@ storiesOf('Textarea', module)
       hideCharacterCount={true}
     />
   ))
+  .add('with a forwardedRef', () => (
+    <Textarea
+      input={inputProps}
+      meta={{}}
+      forwardedRef={inputRef}
+    />
+  ))
```

Comment:
Because this story is using the input directly, we should be able to add a story for a "proper" forwarded ref as well (i.e., `ref={inputRef}`)